lib/ffmpeg/encoderbase.py

```diff
@@ -239,7 +243,7 @@ def check_bitrate(self):
       # acceptable bitrate within 10% of bitrate
       assert(bitrate_gap <= 0.10)
 
-    elif "vbr" == self.rcmode:
+    elif "vbr" == self.rcmode and self.maxFrameSize is None:
```

Comment:
should be `vars(self).get("maxFrameSize", None)`

Reply:
ok